kodilla-spring-web/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'org.springframework.boot' version '2.2.6.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.9.RELEASE'
```

Comment:
jesli to jest modul, nie osobny projekt, to wersje (obie powyzej) sa do wyrzucenia. W przeciwnym wypadku dostaniesz wyjatek, ze wersja jest juz zadeklarowana w parencie

Zweryfikuj czy dziala poprzez odpalenie 
`gradle build `
w roocie projektu

Reply:
nie powinien zostac sam build gradle?

Reply:
ja np w module spring